go.mod

```diff
@@ -119,6 +119,8 @@ require (
 	sigs.k8s.io/yaml v1.6.0
 )
 
+require golang.org/x/sys v0.38.0
```

Comment:
Move this into the block above? idk why go mod tidy doesn't take care of this for us. :-)